docker/entry.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo -e
+
+echo "< Starting Kubestellar container >-------------------------"
+
+mkdir -p /kubestellar-logs
```

Comment:
Isn't this a job for the Dockerfile?

Reply:
I think this is a matter of preference.  I am ok with this living in entry and not in docker file.  It's a toss up honestly.  It can be done in either place.  Not as important 'where' it is done, but that it IS done.